src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static Figure getRandomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
```

https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5#:~:text=The%20Static%20method%20belongs%20to,method%20in%20an%20abstract%20class.

Reply:
Why i should avoid statin in this case?
This is like utility class with have help methods and it doesn't required any object statse.
Methods merely use.